tools/test-proxy/Azure.Sdk.Tools.TestProxy/Store/GitStore.cs

```diff
@@ -553,8 +554,15 @@ public async Task UpdateAssetsJson(string newSha, GitAssetsConfiguration config)
 
                 var currentSHA = (await ParseConfigurationFile(config.AssetsJsonLocation)).Tag;
                 var content = await File.ReadAllTextAsync(config.AssetsJsonLocation);
-                content = content.Replace(currentSHA, newSha);
-
+                if (String.IsNullOrEmpty(currentSHA))
```

Comment:
```suggestion
                if (String.IsNullOrWhiteSpace(currentSHA))
```

Reply:
I may have capitalization wrong.

Your regex is checking for `"Tag":<variable whitespace>"<variable whitespace>"

Reply:
Good catch! Thank you.